src/components/biggive-search/biggive-search.scss

```diff
@@ -0,0 +1,120 @@
+.container {
+  @include standard-font();
+  @include all-round-shadow();
+  width: 400px;
```

Comment:
Does the element need to fix its own width rigidly like this? Why not have the parent control it?